isis/src/base/objs/LidarData/LidarData.cpp

```diff
@@ -50,9 +50,20 @@ namespace Isis {
     m_points.insert(point->GetId(), point);
   }
 
+  /**
+   * Gets a single LidarDataPoint by serial number
+   *
+   * @param serialNumber The serial number of the LidarDataPoint
```

Comment:
This should be the point ID instead of the serial number. Points use IDs, measures and images use serial number.